src/components/Radios.tsx

```diff
@@ -0,0 +1,282 @@
+import { Fieldset, FieldsetProps } from "./Fieldset/Fieldset";
+import PropTypes from "prop-types";
+import React from "react";
+import { Hint, HintProps } from "./Hint";
+import { ErrorMessage, ErrorMessageProps } from "./ErrorMessage";
+import { FormGroup, FormGroupProps } from "./FormGroup";
+import { Label, LabelProps } from "./Label";
+import { Attributes, DataAttributes } from "../helpers/Attributes";
+import classNames from "classnames";
+import "lbh-frontend/lbh/components/lbh-radios/_radios.scss";
+
+export interface RadioButton extends React.AriaAttributes, DataAttributes {
+  /**
+   * Specific id attribute for the radio item. If omitted, then {@link RadiosProps.idPrefix} string will be applied.
+   */
+  id?: string;
+  /**
+   * Value for the radio input.
+   */
+  value?: string;
+  /**
+   * Provide attributes and classes to each radio item label.
+   */
+  label?: LabelProps;
+  /**
+   * Provide hint to each checkbox item.
+   */
+  hint?: HintProps;
+  /**
+   * Divider text to separate radio items, for example the text "or".
+   */
+  divider?: string;
+  /**
+   * If present, radio will be checked.
+   */
+  checked?: boolean;
+  /**
+   * If present content provided will be revealed when the item is checked.
+   * Maps to govuk's conditional prop
+   */
+  childrenWhenChecked?: React.ReactNode;
+  /**
+   * If true, radio will be disabled.
+   */
+  disabled?: boolean;
+}
+
+export interface RadiosProps extends React.AriaAttributes, DataAttributes {
+  /**
+   * Classes to add to the radio container.
+   */
+  className?: string;
+  /**
+   * Name attribute for each radio item.
+   */
+  name: string;
+  /**
+   * Array of {@link RadioButton} objects.
+   */
+  items: RadioButton[];
+  /**
+   * Options for the wrapping {@link Fieldset} component (e.g. legend).
+   */
+  fieldset?: Omit<FieldsetProps, "children">;
+  /**
+   * Options for the {@link Hint} component (e.g. text).
+   */
+  hint?: HintProps;
+  /**
+   * Options for the {@link ErrorMessage} component (e.g. text).
+   */
+  errorMessage?: ErrorMessageProps;
+  /**
+   * Options for the form-group wrapper
+   */
+  formGroup?: Omit<FormGroupProps, "children">;
+  /**
+   * String to prefix id for each checkbox item if no id is specified on each item.
+   * If `idPrefix` is not passed, fallback to using the name attribute instead.
+   */
+  idPrefix?: string;
+
+  onChange?(value: string): void;
+}
+
+export const Radios = (props: RadiosProps): JSX.Element => {
+  const {
+    className,
+    name,
+    items,
+    fieldset,
+    hint: radiosHint,
+    errorMessage,
+    onChange
+  } = props;
+  let formGroup = props.formGroup;
```

Comment:
```suggestion
  const formGroup = props.formGroup || {};
```

This means we don't have to check for existence later.